app/gui/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -130,6 +148,76 @@ fn labeled_button<Icon: ColorableShape>(app: &Application, label: &str) -> Toggl
 
 
 
+// =============================
+// === Context Switch Button ===
+// =============================
+
+/// A button to enable/disable the output context for a particular node. It holds two buttons
+/// internally for each shape, but only once is every shown, based on the execution environment
+/// which sets the global permission for the output context.
+#[derive(Clone, CloneRef, Debug)]
+struct ContextSwitchButton {
+    globally_enabled: Rc<Cell<bool>>,
+    disable_button:   ToggleButton<icon::disable_reevaluation::Shape>,
```

Comment:
Btw, I think `disable_reevaluation` does not represent the purpose of the icon. The node would still be evaluated (but in a different context).